packages/expo-updates/ios/EXUpdates/EXUpdatesAppDelegate.m

```diff
@@ -53,9 +53,14 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     // we just skip in this case.
     return NO;
   }
+  UIWindow *window = application.delegate.window;
+  if ([window.rootViewController.view isKindOfClass:[RCTRootView class]]) {
+    RCTRootView *rootView = (RCTRootView *)window.rootViewController.view;
+    [rootView.bridge invalidate];
```

Comment:
Shouldn't we also set bridge to nil to deallocate it?

Reply:
i'm kinda afraid of `RCTRootView` internally assuming bridge to be nonnull. later the call flow will replace original `RCTRootView` and make it to nil: 
https://github.com/expo/expo/blob/0803ed01bfe56c3911db024457bcbcec79899706/packages/expo-updates/ios/EXUpdates/EXUpdatesAppController.m#L147-L172

bridge will set to nil then when RCTRootView be destroyed.